DataFormats/TrackReco/interface/HitPattern.h

```diff
@@ -1042,6 +1096,21 @@ inline int HitPattern::rpcStationsWithAnyHits() const
     return muonStations(3, -1);
 }
 
+inline int HitPattern::gemStationsWithValidHits() const 
```

Comment:
these 3 methods don't make sense: they conflict with the definition which packs the roll ID into what you are trying to extract as station bits.